src/components/views/Game.js

```diff
@@ -68,6 +68,8 @@ export default function Game() {
     const [roundsPlayed, setRoundsPlayed] = useState("");
     const [team1Size, setTeam1Size] = useState(0);
     const [team2Size, setTeam2Size] = useState(0);
+    const [isBuzzerPressed, setIsBuzzerPressed] = useState(false);
+    const [displayedWord, setDisplayedWord] = useState('');
```

Comment:
This is duplicate information. We already have the displayed word in a field called displayedCard.

Reply:
i need to check if the word is still the same, because the send buzz message returns a card again and updates the displayed card and that is detected as a change o the word and the player can buzer again

Reply:
You can change the comparison to `Card.word !== displayedCard.word`, right?

Reply:
the problem is that displayedCard and Card changes after a buzz also if it is still the same card and as enable the buzzer again if the card changes it always instantly enables the buzzer again